sources/advertisements/VRM New Core/Controllers/VRMTimeoutController.swift

```diff
@@ -0,0 +1,40 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+import PlayerCore
+
+final class VRMTimeoutController {
+    
+    let softTimeoutTimerFactory: () -> Cancellable
```

Comment:
What is `Cancellable`?

Reply:
The protocol that contains the `cancel` method. Our `Timer` and `MockTimer` are extended by this protocol.